### PR TITLE
OSOE-456: Missing NuGet setup in Lombiq.Hosting.MediaTheme

### PR DESCRIPTION
**Pull request by porgabi**
OSOE-456

### CODE REVIEW

File: Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj [hunk 1]
```diff
@@ -1,5 +1,16 @@
...
+  <PropertyGroup>
+    <Title>Lombiq Hosting - Media Theme Deployer for Orchard Core</Title>
+    <Authors>Lombiq Technologies</Authors>
+    <Copyright>Copyright © 2022, Lombiq Technologies Ltd.</Copyright>
+    <Description>Lombiq Hosting - Media Theme Deployer for Orchard Core: A tool to easily export Media Themes.</Description>    
+    <PackageIcon>NuGetIcon.png</PackageIcon>
+    <PackageTags>OrchardCore;Lombiq;AspNetCore;Media;Deployer</PackageTags>
+    <RepositoryUrl>https://github.com/Lombiq/Hosting-Media-Theme</RepositoryUrl>
+    <PackageProjectUrl>https://github.com/Lombiq/Hosting-Media-Theme/blob/dev/Readme.md</PackageProjectUrl>
+  </PropertyGroup>
```
Comment (wAsnk) [inline Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj]: You are missing the licence here

File: Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj [hunk 2]
```diff
@@ -15,4 +26,8 @@
     <PackageReference Include="Newtonsoft.Json" Version="13.0.1" />
   </ItemGroup>
 
+  <ItemGroup>
+    <None Include="NuGetIcon.png" Pack="true" PackagePath="" />
```
Comment (wAsnk) [inline Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj:32]: You are missing the licence here

**Review by wAsnk — CHANGES_REQUESTED** [review]

File: Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj (hunk 1 above)
Comment (porgabi) [reply Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj]: There's no license file for this project (MediaTheme.Deployer) -- do you mean I should add one, just like e.g. MediaTheme.Bridge has one?

Comment (wAsnk) [reply Lombiq.Hosting.MediaTheme.Deployer/Lombiq.Hosting.MediaTheme.Deployer.csproj]: yup

File: Lombiq.Hosting.MediaTheme.Deployer/License.md [hunk 3]
```diff
@@ -0,0 +1,15 @@
+Copyright © 2022, [Lombiq Technologies Ltd.](https://lombiq.com)
```
Comment (wAsnk) [inline Lombiq.Hosting.MediaTheme.Deployer/License.md:1]: Please use the license file given in the 3rd point [here](https://lombiq.atlassian.net/wiki/spaces/DEV/pages/2609840133/Preparing+a+.NET+project+to+be+published+on+NuGet)

**Review by wAsnk — CHANGES_REQUESTED** [review]

**Review by wAsnk — APPROVED** [review]

**Pull request merged**